src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Web3
+description: What is web3 and why is it important?
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, these centralized entities have a stronghold on large swathes of the internet, undemocratically deciding what should and should not be allowed.
+
+Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users.
+Before we talk about web3, let's explore how we got here.
+
+<Divider />
+
+## The early internet {#early-internet}
+
+Most people think of the internet as a continuous pillar of modern life—it was invented and has just existed since. However, the internet most of us know today is quite different from originally imagined. To understand this better, it's helpful to break the internet's short history into loose periods—web 1.0 and web 2.0.
+
+### What is Web 1.0? {#web1}
```

Comment:
```suggestion
### Web1: Read-Only {#web1}
```
I do like the header as a question but I wonder if we can also incorporate the read, read-write, read-write-own into each header. I think it's an excellent & simple framing.